ppr-ui/src/composables/mhrRegistration/useMhrValidations.ts

```diff
@@ -23,24 +28,20 @@ export const useMhrValidations = (validationState: any) => {
 
   /** Reset submission validations to default . */
   const resetAllValidations = (): void => {
-    // Reset your home validations
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.MAKE_MODEL_VALID, false)
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.HOME_SECTION_VALID, false)
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.HOME_CERTIFICATION_VALID, false)
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.REBUILT_STATUS_VALID, false)
-    setValidation(MhrSectVal.YOUR_HOME_VALID, MhrCompVal.OTHER_VALID, false)
-    // Reset submitting party validations
-    setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.SUBMITTER_VALID, false)
-    setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.DOC_ID_VALID, false)
-    setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.REF_NUM_VALID, false)
-    // Reset home owner validations
-    setValidation(MhrSectVal.HOME_OWNERS_VALID, MhrCompVal.OWNERS_VALID, false)
-    // Reset home location validations
-    setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.LOCATION_TYPE_VALID, false)
-    setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.CIVIC_ADDRESS_VALID, false)
-
-    // Reset Review validations
-    setValidation(MhrSectVal.REVIEW_CONFIRM_VALID, MhrCompVal.VALIDATE_STEPS, false)
+    // Gets all sections in model
+    const sections = Object.keys(validationState) as MhrSectVal[]
```

Comment:
Appreciate the explanation of the events. We can also simplify without extra declarations. Either way is fine.

Object.keys(validationState).forEach(section =>
  Object.keys(validationState[section].value).forEach(component =>
    setValidation(section, component, false)
  )
)

Reply:
I am good with simplifying it thanks! 😄